### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by pokryfka**

### CODE REVIEW

File: Sources/AWSXRayRecorder/Emitter.swift [hunk 1]
```diff
@@ -5,7 +5,7 @@ public protocol XRayEmitter {
     func flush(_ callback: @escaping (Error?) -> Void)
 }
 
-public struct XRayNoopEmitter: XRayEmitter {
+public struct XRayNoOpEmitter: XRayEmitter {
```
Comment (ktoso) [inline Sources/AWSXRayRecorder/Emitter.swift:8]: 👍

**Review by ktoso — APPROVED** [review]
Nice example :)

File: Sources/AWSXRayRecorder/Emitter.swift (hunk 1 above)
Comment (pokryfka) [reply Sources/AWSXRayRecorder/Emitter.swift:8]: ;-) previous PR was more juicy, in fact should have been divided int osmaller ones

the examples `gsoc-swift-tracing` look nice, let me follow up on that in https://github.com/slashmo/gsoc-swift-tracing/issues/36

Comment (ktoso) [reply Sources/AWSXRayRecorder/Emitter.swift:8]: :-)

There's a WIP towards defining protocols for tracer instruments here (following open telemetry shape of API mostly): which may be of interest to you btw: https://github.com/slashmo/gsoc-swift-tracing/pull/62 

Thanks in advance, really looking forward to soon trying out xray with it the APIs.

**Pull request merged**